cmd/ebrelayer/txs/parser.go

```diff
@@ -0,0 +1,57 @@
+package txs
+
+// --------------------------------------------------------
+//      Parser
+//
+//      Parses structs containing event information into
+//      unsigned transactions for validators to sign, then
+//      relays the data packets as transactions on the
+//      Cosmos Bridge.
+// --------------------------------------------------------
+
+import (
+	"errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/peggy/cmd/ebrelayer/events"
+	ethbridgeTypes "github.com/cosmos/peggy/x/ethbridge/types"
+)
+
+// ETH : ETH constant specifies a token type of Ethereum
+const (
+	ETH string = "eth"
+)
+
+// ParsePayload : ParsePayload parses a LockEvent struct, packaging the information
+//				  along with a validator address in an EthBridgeClaim msg
```

Comment:
this indent seems to be off?

Reply:
Fixed https://github.com/musnit/peggy/pull/6/commits/664e635e9f0ac62b19dec5828b2346f6a9e80201